docs/content/error/$compile/noident.ngdoc

```diff
@@ -0,0 +1,71 @@
+@ngdoc error
+@name $compile:noident
+@fullName Controller alias is required.
+@description
+
+When using the `bindToController` feature of AngularJS, a directive is required
+to have a Controller identifier. This can be supplied using the "controllerAs"
```

Comment:
Let's standardize on always calling this identifier an **alias** throughout the code and the docs.